src/modules/lua/Makefile

```
@@ -10,10 +10,19 @@ else
 	SHOBJ_LDFLAGS= -shared $(LDFLAGS)
 endif
 
+# BSD variants support
 ifeq ($(uname_S),FreeBSD)
 	SHOBJ_LDFLAGS+= -lm
 endif
 
+ifeq ($(uname_S),OpenBSD)
+	SHOBJ_LDFLAGS+= -lm
+endif
+
+ifeq ($(uname_S),NetBSD)
+	SHOBJ_LDFLAGS+= -lm
+endif
+
```

I believe it would work to search for `$(uname_S)` in a list:

```
ifneq ($(findstring $(uname_S),FreeBSD OpenBSD NetBSD),)
	SHOBJ_LDFLAGS+= -lm
endif
```

Otherwise, else-if in Make is written `else ifeq`:

```suggestion
else ifeq ($(uname_S),OpenBSD)
	SHOBJ_LDFLAGS+= -lm
else ifeq ($(uname_S),NetBSD)
	SHOBJ_LDFLAGS+= -lm
endif
```

---

Or better, so we also cover odd birds like DragonflyBSD:

```
ifeq ($(findstring BSD,$(uname_S)),BSD)
	SHOBJ_LDFLAGS+= -lm
endif
```

---

Nevermind, apparently `uname -s` is "DragonFly" on DragonflyBSD.